apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1091,22 +1091,24 @@ send_response(S, Type, Response) ->
     Msg = aec_peer_messages:serialize_response(Type, Response),
     send_msg(S, response, Msg).
 
-%% If the message is more than 65533 bytes it won't fit in a single Noise
-%% message and we need to fragment it.
+%% If the message is more than 65535 - 2 - 16 bytes (2 bytes for the length and
```

Comment:
The 16 comes from [here](https://github.com/aeternity/enoise/blob/71aac15db7b934e0aecac87ff0175abf40aaa2f2/src/enoise_connection.erl#L177)? Maybe in a next PR the hardcoded numbers may be replaced with calls to some functions exported by enoise_connection for readability.

Reply:
Yes, good suggestion except then I can't use it as a guard 🙄 Perhaps some local constants with a reference to the noise protocol? (This 2 byte length is a top level protocol detail.)